pythonx/jedi_vim.py

```diff
@@ -148,6 +157,36 @@ def wrapper(*args, **kwargs):
     return func_receiver
 
 
+last_force_python_error = None
+
+
+def get_environment():
+    global last_force_python_error
+
+    force_python_version = vim_eval("g:jedi#force_py_version")
+    environment = jedi.api.environment.get_cached_default_environment()
```

Comment:
Could be skipped with `!= "auto"` - only needed in case on an exception then.

Reply:
I changed it.